backend/tests/Feature/Machine/SetMachineStateTest.php

```
@@ -0,0 +1,95 @@
+            ->assertRedirect();
+
+        $this->assertDatabaseHas('workstation_states', [
+            'workstation_id' => $this->workstation->id,
+            'state' => 'CLEANING',
+            'source' => 'manual',
+        ]);
+    }
+
+    public function test_operator_cannot_set_state_for_a_workstation_on_another_line(): void
+    {
+        $otherLine = Line::factory()->create();
```

PHPStan (2.2.2)

[error] Call to an undefined static method App\Models\Line::factory().


(staticMethod.notFound)